packages/react/src/components/DataTable/components/DataTableFooterColumn/DataTableFooterColumn.tsx

```diff
@@ -9,19 +10,27 @@ export function DataTableFooterColumn<TData extends RowData, TVaue>(
 ) {
   const { header, table, footerProps, index } = props;
   const { column } = header;
-  const { data, keyword, callBackFunc } = footerProps;
+  const { data, keyword, callBackFunc, enableSticky } = footerProps;
 
   const styles: React.CSSProperties = getColumnLayoutStyles(table, column);
+  const sticky: React.CSSProperties = {
+    position: 'sticky',
+    bottom: '0px',
+    zIndex: 0,
+  };
 
   return (
     <th
       key={header.id}
       className={cx('manifest-table__cell', 'manifest-table__cell--header', {
-        'manifest-table__cell--sticky-header': table.options.enableStickyHeader,
-        'manifest-table__cell--pinned': '',
+        // 'manifest-table__cell--sticky-header': table.options.enableStickyHeader,
+        'manifest-table__cell--pinned': column.getIsPinned(),
```

Comment:
please remove instead of commenting out if it's not needed

Reply:
Fixed these